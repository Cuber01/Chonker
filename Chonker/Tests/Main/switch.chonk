switch(10)
{
    
    case > 1:
    {
        puts "minor gaming";
    }

    case 2:
    {
        puts "no gaming >:(";
    }

    default:
    {
        puts "gaming";
    }

}

switch(true)
{

    case true:
    {
        puts "+1 byczq";
    }

    case false:
    {
        puts "Fake news!!!";
    }

}

switch(null)
{

    case true:
    {
        puts "+1 byczq";
    }

    case false:
    {
        puts "Fake news!!!";
    }

    default:
    {
        puts "Coping";
    }

}


