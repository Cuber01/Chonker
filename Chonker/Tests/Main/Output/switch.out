minor gaming
+1 byczq
Coping
